### DISCUSSION TITLE
Welcome to Diplomacy Discussions!

### DISCUSSION THREAD

**Discussion by zijistark** [DiplomacyTeam/Bannerlord.Diplomacy#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iwanadiefast** [comment]
Some feedback actually. there's something weird happening, where this mod in particular prevents the game from loading a saved game.
most other features function just fine (except declaring peace or war against another faction even though you're a tier 2 mercenary and some others that i don't really remember but are basically "yea it's there but just don't do it mate")
build like a charm in vs19
